tests/test_deserialization.py

```diff
@@ -284,18 +283,29 @@ class MyBoolean(fields.Boolean):
     ])
     def test_boolean_field_deserialization_with_custom_truthy_values_invalid(
             self, in_val):
-        class MyBoolean(fields.Boolean):
-            truthy = set(['yep'])
-        field = MyBoolean()
+        field = fields.Boolean(truthy=('yep',))
```

Comment:
I agree. @zwack Once the above change is implemented, this should be good to go.